.github/workflows/openshift-nightly.yml

```diff
@@ -26,6 +30,22 @@ jobs:
         run: |
           make build-osm
           go test ./tests/e2e -test.v -ginkgo.v -ginkgo.progress -ginkgo.skip="\bHTTP ingress\b" -ginkgo.skip="\bUpgrade\b" -test.timeout 180m -deployOnOpenShift=true
```

Comment:
sidebar: is upgrade test passing now with this change? https://github.com/openservicemesh/osm/pull/3835

Reply:
We're going to follow the same logic as the PR gates and only run the upgrade tests on the release branch.

I've created a tracking issue here: https://github.com/openservicemesh/osm/issues/3852